app/assets/stylesheets/application.css.scss

```diff
@@ -86,7 +86,9 @@ nav .brand-logo {
 
 @font-face {
   font-family: 'Jungle';
-  src: url('/assets/JungleClean.ttf')
+  src: font-url( asset-path('JungleClean.ttf') );
```

Comment:
Expected 0 spaces after `(` instead of ` `<br>Expected 0 spaces before `)` instead of ` `<br>Prefer double-quoted strings